### PR TITLE
BXMSDOC-8323 Adding support to switch to Elytron security subsystem

### PR DESCRIPTION
**Pull request by briandooley**
Jira: [BXMSDOC-8323](https://issues.redhat.com/browse/BXMSDOC-8323).
[RHPAM Rendering](http://file.ork.redhat.com/~bdooley/BXMSDOC-8323-RHPAM/#patches-applying-proc_patching-upgrading).
[RHDM Rendering](http://file.ork.redhat.com/~bdooley/BXMSDOC-8323-RHDM/#patches-applying-proc_patching-upgrading).

### CODE REVIEW

**Review by mareknovotny — APPROVED** [review]
lgtm

**Review by tomasdavidorg — APPROVED** [review]
Looks good to me. Thanks @briandooley

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 1]
```diff
@@ -130,6 +130,51 @@ $ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION_LONG}.GA-maven-repository/ma
 ====
 You can remove the `/tmp/rhbaMavenRepoUpdate` folder after you complete the update.
 ====
+
+. If you are changing {PRODUCT} from using property-based user storage to file-based user storage, apply these file updates and configuration updates.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]:
```suggestion
. If you are changing {PRODUCT} from using property-based user storage to file-based user storage, complete the following steps:
```

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 2]
```diff
@@ -130,6 +130,51 @@ $ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION_LONG}.GA-maven-repository/ma
 ====
 You can remove the `/tmp/rhbaMavenRepoUpdate` folder after you complete the update.
 ====
+
+. If you are changing {PRODUCT} from using property-based user storage to file-based user storage, apply these file updates and configuration updates. 
++
+Navigate to the temporary directory where you extracted the `{PRODUCT_INIT}-$VERSION-update.zip` file and run the `kie-fs-realm-patch` script in the following format:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]:
```suggestion
.. Navigate to the directory where you extracted the `{PRODUCT_INIT}-$VERSION-update.zip` file and run one of the following commands to apply the `kie-fs-realm` patch:
```

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 3]
```diff
@@ -130,6 +130,51 @@ $ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION_LONG}.GA-maven-repository/ma
...
++
+On Linux or Unix-based systems:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]:
```suggestion
* On Linux or Unix-based systems:
```

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 4]
```diff
@@ -130,6 +130,51 @@ $ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION_LONG}.GA-maven-repository/ma
...
++
+[source,subs="attributes+"]
+----
+$ ./elytron/kie-fs-realm-patch.sh ~/$JBOSS_HOME/standalone/configuration/kie-fs-realm-users/
+----
++
+On Windows:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]:
```suggestion
* On Windows:
```

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 5]
```diff
@@ -130,6 +130,51 @@ $ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION_LONG}.GA-maven-repository/ma
...
++
+[source,subs="attributes+"]
+----
+$ ./elytron/kie-fs-realm-patch.bat $JBOSS_HOME/standalone/configuration/kie-fs-realm-users/
+----
++
+Navigate to the `$JBOSS_HOME` directory and run the following commands:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]:
```suggestion
.. Navigate to the `$JBOSS_HOME` directory and run the following commands:
```

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 6]
```diff
@@ -130,6 +130,51 @@ $ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION_LONG}.GA-maven-repository/ma
...
++
+On Linux or Unix-based systems:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]:
```suggestion
* On Linux or Unix-based systems:
```

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 7]
```diff
@@ -130,6 +130,51 @@ $ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION_LONG}.GA-maven-repository/ma
...
+----
+./bin/standalone.sh --admin-only -c standalone-full.xml
+----
++
+[source,subs="attributes+"]
+----
+./bin/jboss-cli.sh --connect --file={PRODUCT_INIT}-$VERSION-update/rhpam-7.12.0-update/elytron/add-kie-fs-realm.cli
+----
++
+On Windows:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]:
```suggestion
* On Windows:
```

**Comment by briandooley** [comment]
@emmurphy1 I have committed your suggestions. Can I consider this approved?

**Pull request merged**